### DISCUSSION TITLE
Introducing Android preview 3

### DISCUSSION THREAD

**Discussion by jp-weber** [2fast-team/2fast#177] [Announcements]
Third public Android preview with some limitations:
- only loading of a data file possible

<hr /><em>This discussion was created from the release <a href='https://github.com/2fast-team/2fast/releases/tag/Android-preview-3'>Introducing Android preview 3</a>.</em>

**Comment by maxii1996** [comment]
Thank you so much ^^

I've been waiting for this for a long time! Finally I don't have to turn on my PC at dawn to log in to my networks and I can do it from my phone!
